pages/queries.rst

```diff
@@ -74,12 +74,12 @@ See `wildcard and regexp queries <https://www.elastic.co/guide/en/elasticsearch/
 **Fuzziness:** You can search for similar but not equal terms::
 
   ssh logni~
-  source:exmaple.org~
+  source:example.org~
```

Comment:
Would you be ok w/ including ("example is intentionally misspelled below") or something?
(That wouldn't be a change I'd make in this PR, I will drop these two commits when I'm next at my personal workstation.)